.travis.yml

```diff
@@ -24,3 +25,6 @@ jobs:
       script: npm run test
     - stage: coverage
       script: npm run coverage
+    - stage: release
+      if: type != pull_request
+      script: npm run build && npx semantic-release
```

Comment:
Does this only apply to the `master` branch? If I ever create a new branch and push to it before even making a PR, do you know if it's going to create a new release?